Routing/AnnotatedRouteControllerLoader.php

```diff
@@ -80,11 +80,12 @@ protected function getGlobals(\ReflectionClass $class)
      */
     protected function getDefaultRouteName(\ReflectionClass $class, \ReflectionMethod $method)
     {
-        $routeName = parent::getDefaultRouteName($class, $method);
+        $classMethodParsed = preg_replace('/([a-z])([A-Z])/', '$1_$2', $class->name.'_'.$method->name);
+        $routeName = strtolower(str_replace('\\', '_', $classMethodParsed));
```

Comment:
This breaks tests as it does not deal with multi-URLs for a single controller like done in the parent class.